### PR TITLE
 AC Automated Test - LRAC-8326, LRAC-8327 | master

### PR DESCRIPTION
**Pull request by 91nilo**
https://issues.liferay.com/browse/LRAC-8326
https://issues.liferay.com/browse/LRAC-8327

### CODE REVIEW

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by 91nilo** [comment]
Removed "ignore = true"

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by 91nilo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesCohortAnalysis.testcase [hunk 1]
```diff
@@ -57,11 +57,45 @@ definition {
...
 	@priority = "3"
 	test AssertSitesCohortAnalysisSortAnonymousVisitors {
 
-		// TODO LRAC-8327 AssertSitesCohortAnalysisSortAnonymousVisitors
+		task ("Get site assigned property name") {
+			var propertyName = ACDXPSettings.getAssignedPropertyName(siteName = "Site Name");
+		}
+
+		task ("Logout from DXP") {
+			ACUtils.launchDXP();
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesCohortAnalysis.testcase:66]: Hi @91nilo, As you are already in DXP, I believe that it is not necessary to use this macro, just logout as you do in the following macro

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesCohortAnalysis.testcase [hunk 2]
```diff
@@ -57,11 +57,45 @@ definition {
...
+
+			ACProperties.switchProperty(propertyName = "${propertyName}");
+		}
+		task ("Assert that the cohort analysis card shows anonymous visitors after their first views") {
+			Select(
+				locator1 = "ACSites#COHORT_ANALYSIS_SELECT_VISITOR_TYPE",
+				value1 = "Anonymous Visitors");
+			
+			AssertTextEquals(
+				locator1 = "ACSites#COHORT_ANALYSIS_DAY_ZERO_PERCENTAGE",
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesCohortAnalysis.testcase:94]: Hi @91nilo, I'm not able to find the macro on ACSites or on the master, could you check if it hasn't been removed?

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by 91nilo** [comment]
Hi @ViniLopes20 i did the changes, but if you think there is anything else that i could improve, just tell me. Thanks!

**Comment by alexcandidos** [comment]
Hi @91nilo , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**